alacritty_config_derive/tests/config.rs

```diff
@@ -189,6 +194,30 @@ fn replace_derive() {
     assert_eq!(test.nesting.newtype, NewType(9));
 }
 
+#[test]
+fn replace_derive_using_alias() {
+    let mut test = Test::default();
+
+    assert_ne!(test.field1, 9);
+
+    let value = toml::from_str("noalias=9").unwrap();
```

Comment:
I'm not sure why this field has the alias "noalias", but would probably best to change that. It makes no sense to me.